### ISSUE TITLE
[SCREENREADER]: Current page is not being announced in page numbers, and some page links are not keyboard focusable

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#902]
## Issue
The page numbers are not announcing the current page to assistive device users. They're also not announcing themselves as links because we don't have an `href` attribute on each link. I'd like to improve these links by refactoring the markup a bit. Screenshot and code example attached below.

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/713

## Acceptance Criteria
* As a screen reader user, I want the current page of results to announce as the current item.
* I also want the other page result links to announce themselves as links.

## Environment
* Windows 10
* IE11 + JAWS
* https://staging.va.gov/find-locations/ (Searched for "Salt Lake City", Community Care, Nurse Practitioner)

## Possible Fixes (optional)
The HTML example below is truncated, but should offer a good outline of what we'd like to accomplish for the pagination links.

```html
<span class="...">
  <a href="#" class="va-button-link" id="first-click" aria-label="First page">
    <span class="...">First</span>
    </span>
  </a>
</span>

<span class="va-pagination-prev">
  <a href="#" class="va-button-link" id="pager-previous-click" aria-label="Load previous page">Prev</a>
</span>

<!-- I'd like to add real URLs so the pages announce as links -->
<div class="va-pagination-inner">
  <ul>
    <li
      aria-current="true"
	  aria-label="Page 1"
	  class="va-pagination-active"
    >1</li>
  	<li>
      <a
        aria-label="Load page 2"
        class="va-pagination-active"
        href="real-href-of-links"
      >2</a>
    </li>
    <li>
      <a
        aria-label="Load page 3"
        class="va-pagination"
        href="real-href-of-links"
      >3</a>
    </li>
  </ul>
</div>

...
<!-- Next link should be spelled out, and Last link too -->
```

## WCAG or Vendor Guidance (optional)
* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)
* [Using the aria-current attribute - Tink.UK](https://tink.uk/using-the-aria-current-attribute/)

## Screenshots or Trace Logs
![Screen Shot 2019-08-09 at 4.21.08 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/2b8d4d1f-64eb-4219-869d-4b7af7904b57)

**Comment by jefflbrauer** [comment]
@1Copenut This fix is now on staging. please go ahead and retest.
https://staging.va.gov/outreach-and-events

**Comment by 1Copenut** [comment]
@jefflbrauer I retested on Staging and the current page is being announced properly. Can now also focus the links with keyboard.

We have a `Pagination` component in the [VA design system](https://design.va.gov/components/pagination) that could use this same type of refactoring. I'll put in a separate ticket for that as a sitewide improvement based on our findings here.

Moving this one to close.
